src/components/Header/Header.test.js

```diff
@@ -0,0 +1,11 @@
+import Header from "./Header";
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+describe('Header', () => {
+
+    test('Should render Header', () => {
```

Comment:
I think your test is not correct, because each test must check something and have expect and actual result, or if you want to test render component you can do snapshot

pls fix this test

https://jestjs.io/docs/ru/snapshot-testing